.github/actions/fuse_compat/action.yml

```diff
@@ -30,6 +30,8 @@ runs:
           bash ./tests/fuse-compat/test-fuse-compat.sh 1.1.39 base
           bash ./tests/fuse-compat/test-fuse-compat.sh 1.1.46 base
           bash ./tests/fuse-compat/test-fuse-compat.sh 1.2.306 rbac
+          bash ./tests/fuse-compat/test-fuse-forward-compat.sh 1.2.306 rbac
```

Comment:
```
 === Start ./bins/1.2.306/bin/databend-query databend-query...
databend config path: old_config/scripts/ci/deploy/config/databend-query-node-1.toml
... connecting to :3307
... connecting to :3307
... connecting to :3307
... connecting to :3307
Databend Query start failure, cause: InvalidReply. Code: 2005, Text = Cannot deserialize RoleInfo from bytes cause: serde_json::error::Error: expected value at line 2 column 2.
```

Because current query version will serialize new role info as pb.

306 query version only support deserialize role info as json. So current query version it not forward compat with 1.2.306. That is known. We need prove it's forward compat with 307.